docs/tools/vdb_table/data/databricks.json

```diff
@@ -0,0 +1,188 @@
+{
+    "name": "Databricks Vector Search",
+    "links": {
+      "docs": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "github": "",
+      "website": "https://www.databricks.com/product/machine-learning/vector-search",
+      "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/514",
+      "poc_github": "kobindra",
+      "slug": "databricks"
+    },
+    "oss": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Proprietary",
+      "source_url": "https://www.databricks.com/legal/mcsa",
+      "comment": ""
+    },
+    "dev_languages": {
+      "value": [
+        ""
+      ],
+      "source_url": "",
+      "comment": ""
+    },
+    "vector_launch_year": 2024,
+    "metadata_filter": {
+      "support": "full",
+      "source_url": "https://www.databricks.com/blog/announcing-mosaic-ai-vector-search-general-availability-databricks",
+      "comment": ""
+    },
+    "hybrid_search": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": ""
+    },
+    "facets": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "geo_search": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "multi_vec": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "sparse_vectors": {
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "bm25": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#how-does-mosaic-ai-vector-search-work",
+      "comment": "Okapi BM25"
+    },
+    "full_text": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html",
+      "comment": ""
+    },
+    "embeddings_text": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": ""
+    },
+    "embeddings_image": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": "With self-managed Delta Sync"
+    },
+    "embeddings_structured": {
+      "support": "full",
+      "source_url": "https://docs.databricks.com/en/generative-ai/vector-search.html#options-for-providing-vector-embeddings",
+      "comment": "With self-managed Delta Sync"
+    },
```

Comment:
would be good to link to a model catalog like https://docs.databricks.com/en/generative-ai/external-models/index.html. looks good other than that

Reply:
"Delta table that contains data in text format" - this is not structured data, I don't think this column is ticked @kobindra - for illustration, structured data could be numbers, categorical variables/enums, graph edges/foreign keys into other tables - that kind of stuff. Can Databricks Vector Search turn this kind of stuff into embeddings out of the box?

Reply:
As of today, the choice of models for automatically generating embeddings for customers is restricted to text based models (for now, in the future we will expand this to other models). Of course, we also have a vector in / vector out interface, and it supports any kind of embedding model the customer wants to use. About 40% of all indexes in Vector Search are self-managed embeddings generated by customers.

So if the question of supporting these various types of embeddings is whether we generate the embeddings for the user, then we currently don't, and I'm happy to adjust the level of support to be consistent with the standard for accepting user-generated embeddings of any data type. Is "partial" the right level of support for that?

@dhruv-anand-aintech -- I think this applies to your question about which models we support w/r/t image embeddings as well